download_and_install/ola_on_raspberry_pi.md

```diff
@@ -0,0 +1,69 @@
+OLA on Raspberry Pi
+===================
+
+This tutorial describes how to get OLA running on the
+[Raspberry Pi](https://www.raspberrypi.org/). The procedure described here is designed to get OLA up and running as
+easily as possible. There is plenty of information on
+the [Raspberry Pi Docs](https://www.raspberrypi.org/documentation/).
+The [Raspberry Pi Forum](https://www.raspberrypi.org/forums/) is a good place to ask for help on Raspberry Pi specific
+issues.
+
+Getting Started
+===============
+
+See the official [setup guide](https://www.raspberrypi.org/documentation/setup/) for hardware requirements.
+
+- A monitor is not strictly required to use OLA, but one can be helpful for debugging purposes.
+- If you plan on using a USB DMX interface that does not use its own power supply, you will also need a powered USB hub.
+  The Raspberry Pi hardware cannot supply enough power over USB to meet the needs of most USB DMX interfaces.
+- A wired network connection is required when using OLA with any network protocols. Remember, DMX over WiFi is not Wireless DMX!
```

Comment:
```suggestion
- A wired network connection is recommended when using OLA with any network protocols. Remember, DMX over WiFi is not Wireless DMX!
```

It will work and some people do use it! Or maybe highly recommended.

Reply:
That's terrifying to this NYC person, where if the WiFi spectrum was physical none of us could breath :grinning:. I might err on the side of strongly recommending a wired connection for production use.

Reply:
Yeah that sounds good. I suspect a common usage may be as a focus remote, if their mobile phone app offers better features than the mobile keypad of OLA (e.g. multiple universes), they might want to say have OLA as DMX over Ethernet to DMX bridge and the phone sending to that.

I suspect for some, budget may come into it too, if you're low budget e.g. am-dram, you're probably less likely to be in a place with such dense levels of WiFi if you were using it for a mobile prop.